documentation/modules/module_doc_template.md

```diff
@@ -4,7 +4,9 @@ functioning in 5+ years, so giving links or specific examples can be VERY helpfu
 
 ## Vulnerable Application
 
-Instructions to get the vulnerable application.  If applicable, include links to the vulnerable install files, as well as instructions on installing/configuring the environment if it is different than a standard install. Much of this will come from the PR, and can be copy/pasted.
+Instructions to get the vulnerable application.  If applicable, include links to the vulnerable install
```

Comment:
There is still an extra space before "If applicable", can this be removed?

Reply:
Us old people double space after periods :) but you're right it isn't consistent in there